src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -30,18 +30,17 @@
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.6.1" />
     <PackageReference Include="Grpc.Core" Version="1.18.0" />
-    <PackageReference Include="Microsoft.ApplicationInsights" Version="2.7.2" />
```

Comment:
Removing this explicit reference so we'll naturally use the one that comes with our AppInsights package.